changelog.md

```diff
@@ -1,3 +1,142 @@
+ <a name="4.0.0-alpha.0"></a>
+# 4.0.0-alpha.0 (2018-10-31)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v3.2.0...v4.0.0-alpha.0)
+
+## New Contributors!
+Thanks to @carlesnunez and @justin0022 for their first contributions!
+
+Thanks to @CodeDem for improving how the image-aspect-ratio audit handles small images!
+
+Huge thanks to myself (@Hoten) for joining Lighthouse! Go me!
```

Comment:
Back in my day we just let @paulirish release, and he never mentioned if you even existed. :eyes: